tsconfig.json

```
@@ -0,0 +1,12 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "target": "ES5",
```

All browser support modern JavaScript. We should leave it up to the users of this lib to decide what level of JavaScript they want to consume.

```suggestion
    "module": "es2020",
    "target": "es2019",
```